mymangomail.com.email.json

```diff
@@ -0,0 +1,83 @@
+{
+  "providerId":"mymangomail.com",
+  "providerName":"Mango Mail",
+  "serviceId":"email",
+  "serviceName":"Email Setup",
+  "sharedServiceName": true,
+  "version":1,
+  "logoUrl":"https://mymangomail.com/assets/mango.svg",
+  "description":"Set up the necessary records to enable Mango Mail email hosting",
+  "variableDescription":"verification: Unique identifier to validate domain ownership",
+  "syncBlock": false,
+  "syncPubKeyDomain": "mymangomail.com",
+  "records":[
+
+    {
+      "groupId":"setup",
+      "type":"TXT",
+      "host":"@",
+      "data":"mail-host-verify=%verification%",
+      "ttl":3600
+    },
+
+    {
+      "groupId":"setup",
+      "type":"SPFM",
+      "host":"@",
+      "spfRules":"v=spf1 include:spf.mymangomail.com -all",
+      "ttl":3600
+    },
+
+    {
+      "groupId":"setup",
+      "type":"TXT",
+      "host":"_dmarc",
+      "data":"v=DMARC1; p=quarantine;",
+      "ttl":3600

```

Comment:
For DMARD it's recommended to set `txtConflictMatchingMode`=`All`

Reply:
Fixed this too.